server/portfolio/portfolio/blueprints/api.py

```diff
@@ -41,6 +43,8 @@ async def get_assets():
         for response in responses:
             results.append(response)
 
+        convert_assets_value_to_base_currency(user_data['base_currency'], results)
+
```

Comment:
Please look at the logic changes that have been made in Portfolio in #206 . I think it should look something like this
```suggestion
        balances = convert_assets_value_to_base_currency(user_data['base_currency'], responses[0])
        holdings = convert_assets_value_to_base_currency(user_data['base_currency'], responses[1])
```

Reply:
done in commit 6d4a61d01110eec44b145a70bc7b1577a0e4c996